sale_stock_picking_blocking/models/sale_order.py

```diff
@@ -45,6 +45,17 @@ def action_remove_delivery_block(self):
             order_to_unblock.action_done()
         return True
 
+    @api.model_create_multi
+    def create(self, vals_list):
```

Comment:
@ntsirintanis Could you try adding `precompute=True` to the field instead ?

Reply:
@rousseldenis thank you very very much, it works. But why?